VERSION

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
Should we use the .99 convention to show that we are between tagged releases? (Do we tag releases?)

Reply:
Good idea, done